include/netlink/route/link/can.h

```diff
@@ -51,6 +51,19 @@ extern int rtnl_link_can_get_ctrlmode(struct rtnl_link *, uint32_t *);
 extern int rtnl_link_can_set_ctrlmode(struct rtnl_link *, uint32_t);
 extern int rtnl_link_can_unset_ctrlmode(struct rtnl_link *, uint32_t);
 
+extern int rtnl_link_can_get_data_bt_const(struct rtnl_link *,
+														 struct can_bittiming_const *);
+extern int rtnl_link_can_get_data_bittiming(struct rtnl_link *,
+														  struct can_bittiming *);
+extern int rtnl_link_can_set_data_bittiming(struct rtnl_link *,
+														  struct can_bittiming *);
+
+extern int rtnl_link_can_get_data_bitrate(struct rtnl_link *, uint32_t *);
+extern int rtnl_link_can_set_data_bitrate(struct rtnl_link *, uint32_t);
+
+extern int rtnl_link_can_get_data_sample_point(struct rtnl_link *, uint32_t *);
+extern int rtnl_link_can_set_data_sample_point(struct rtnl_link *, uint32_t);
```

Comment:
why are there here a few getters/setters for individual fields, when above we have the "get-data/set-data" to set them all? Aren't they redundant?

And about having "get-data/set-data", it requires that `struct can_bittiming(_const)` doesn't ever change the struct layout and size. I guess, that is guaranteed by kernel user space API?

Reply:
I'll drop those accessors (at least for now).